File: src/main/scala/grp/faq/entity/Praise.scala
package grp.faq.entity

import scala.beans.BeanProperty

class Praise() {
  @BeanProperty var cPraiseid: Long = 0L
  @BeanProperty var cUid: Int = 0
  @BeanProperty var cPratype: Int = 0
  @BeanProperty var cCommonId: Long = 0L
  @BeanProperty var cIsDelete: Int = 0

  def this(cPraiseid: Long, cUid: Byte, cPratype: Byte, cCommonId: Long, cIsDelete: Byte) {
    this()
    this.cPraiseid = cPraiseid
    this.cUid = cUid
    this.cPratype = cPratype
    this.cCommonId = cCommonId
    this.cIsDelete = cIsDelete
  }
}

File: src/main/scala/grp/faq/entity/Praise.java
package grp.faq.entity;

public class Praise {
    public Long cPraiseid;

    public Byte cUid;

    public Byte cPratype;

    public Long cCommonId;

    public Byte cIsDelete;

    public Praise(Long cPraiseid, Byte cUid, Byte cPratype, Long cCommonId, Byte cIsDelete) {
        this.cPraiseid = cPraiseid;
        this.cUid = cUid;
        this.cPratype = cPratype;
        this.cCommonId = cCommonId;
        this.cIsDelete = cIsDelete;
    }

    public Praise() {
        super();
    }

    public Long getcPraiseid() {
        return cPraiseid;
    }

    public void setcPraiseid(Long cPraiseid) {
        this.cPraiseid = cPraiseid;
    }

    public Byte getcUid() {
        return cUid;
    }

    public void setcUid(Byte cUid) {
        this.cUid = cUid;
    }

    public Byte getcPratype() {
        return cPratype;
    }

    public void setcPratype(Byte cPratype) {
        this.cPratype = cPratype;
    }

    public Long getcCommonId() {
        return cCommonId;
    }

    public void setcCommonId(Long cCommonId) {
        this.cCommonId = cCommonId;
    }

    public Byte getcIsDelete() {
        return cIsDelete;
    }

    public void setcIsDelete(Byte cIsDelete) {
        this.cIsDelete = cIsDelete;
    }
}